build_tools/benchmarks/common/benchmark_presentation.py

```diff
@@ -80,6 +181,41 @@ def aggregate_all_benchmarks(
   return aggregate_results
 
 
+def collect_all_compilation_metrics(
+    compile_stats_files: Sequence[str],
+    expected_pr_commit: Optional[str] = None) -> Dict[str, CompilationMetrics]:
+  """Collects all compilation statistics in the given files.
+
+    Args:
+      compile_stats_files: A list of JSON files, each can be decoded as a
+        CompilationResults.
+      expected_pr_commit: An optional Git commit SHA to match against.
+
+    Returns:
+      A dict of benchmark names to CompilationMetrics.
+  """
+  compile_metrics = {}
+
+  for compile_stats_file in compile_stats_files:
+    with open(compile_stats_file) as f:
+      file_results = CompilationResults.from_json_object(json.load(f))
+
+    if (expected_pr_commit is not None) and \
+            (file_results.commit != expected_pr_commit):
```

Comment:
Use parens instead of a line continuation

```suggestion
    if ((expected_pr_commit is not None) and
            (file_results.commit != expected_pr_commit)):
```